gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/PlanServiceImpl.java

```diff
@@ -189,29 +190,43 @@ public PlanEntity create(final ExecutionContext executionContext, final NewPlanE
         }
     }
 
+    private void validateTags(Set<String> tags, ApiEntity api) {
```

Comment:
Validation should be made using a dedicated service as for any others attributes see io.gravitee.rest.api.service.v4.impl.validation.ApiValidationServiceImpl